packages/presentation/lib/screen/home/bloc/home_bloc.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:math';
+
+import 'package:domain/usecase/generate_number_usecase.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:domain/usecase/check_number_usecase.dart';
+
+part 'home_event.dart';
+
+part 'home_state.dart';
+
+class HomeBloc extends Bloc<HomeEvent, HomeState> {
+  final blocCheckUseCase = CheckNumberUseCase();
+  final blocGenerateUseCase = GenerateNumberUseCase();
```

Comment:
better to inject use cases in to constructor

Reply:
Why u dislike the way I used?

Reply:
dependency injection, but yes, @vladimir-hodaryonok-idf is right, pass it to constructor please

Reply:
corrected